test/testtokenize.cpp

```diff
@@ -73,6 +73,7 @@ class TestTokenizer : public TestFixture {
         TEST_CASE(tokenize32);  // #5884 (fsanitize=undefined: left shift of negative value -10000 in lib/templatesimplifier.cpp:852:46)
         TEST_CASE(tokenize33);  // #5780 Various crashes on valid template code
         TEST_CASE(tokenize34);  // #8031
+        TEST_CASE(tokenize35);  // #8091
```

Comment:
Since you are testing Tokenizer::simplifyNamespaceStd() I want that you add this testing in TestTokenizer::simplifyNamespaceStd().